waspc/test/Generator/WebAppGeneratorTest.hs

```diff
@@ -27,7 +27,11 @@ spec_WebAppGenerator = do
               [ AS.Decl.makeDecl
                   "TestApp"
                   AS.App.App
-                    { AS.App.title = "Test App",
+                    {
+                      AS.App.title = "Test App",
+                      AS.App.wasp = AS.App.Wasp {
+                        AS.App.version = "0.6.0.0"
+                      },
```

Comment:
Why is this `AS.App.version` and not `AS.App.Wasp.version`?

Reply:
I see it working as:
```haskell
AS.App.wasp =
  AS.Wasp.Wasp
    { AS.Wasp.version = "^" ++ show WV.waspVersion
    },
```